pandas/core/arrays/base.py

```diff
@@ -39,6 +39,30 @@ class ExtensionArray:
 
     .. versionadded:: 0.23.0
 
+    Attributes
+    ----------
+    dtype
+    nbytes
+    ndim
+    shape
+
+    Methods
+    -------
+    argsort
+    astype
+    copy
+    dropna
+    factorize
+    fillna
+    isna
+    ravel
+    repeat
+    searchsorted
+    shift
+    take
+    unique
+    _values_for_factorize
```

Comment:
Should we include more of the methods of the EA interface that start with an underscore? (I think that was the idea of https://github.com/pandas-dev/pandas/issues/24067)

Reply:
@jorisvandenbossche Ok, I've added all the methods. That raised some more Docstring validation errors which are resolved now.
Thank you for reviewing!